sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -770,6 +770,39 @@ describe("LocalWorkspace", () => {
 
         await stack.destroy();
     });
+    it(`renames a stack`, async () => {
+        const program = async () => {
+            class MyResource extends ComponentResource {
+                constructor(name: string, opts?: ComponentResourceOptions) {
+                    super("my:module:MyResource", name, {}, opts);
+                }
+            }
+            new MyResource("res");
+            return {};
+        };
+        const projectName = "inline_node";
+        const stackName = fullyQualifiedStackName(getTestOrg(), projectName, `int_test${getTestSuffix()}`);
+        const stack = await LocalWorkspace.createStack(
+            { stackName, projectName, program },
+            withTestBackend({}, "inline_node"),
+        );
+
+        const renamed = stackName + "_renamed";
+        const renameRes = await stack.rename({ stackName: renamed });
```

Comment:
Is it worth testing the `onOutput`/`onEvent` as well?  (Just noticed it in the codecov output)

Reply:
It turns out the event log doesn't actually exist for this command, so I removed it - added a test for `onOutput`, though.